tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -47,20 +47,23 @@ jobs:
       script: |-
         export ROCM_HOME=/opt/rocm
         python tools/ci_build/build.py \
-        --config RelWithDebInfo \
-        --enable_training \
-        --enable_training_torch_interop \
-        --mpi_home /opt/ompi \
-        --use_rocm \
-        --rocm_version=5.2.3 \
-        --rocm_home /opt/rocm \
-        --nccl_home /opt/rocm \
-        --update \
-        --build_dir ./build \
-        --build \
-        --parallel 8 \
-        --build_wheel \
-        --skip_tests
+          --config RelWithDebInfo \
+          --enable_training \
+          --enable_training_torch_interop \
+          --mpi_home /opt/ompi \
+          --cmake_extra_defines \
+              CMAKE_HIP_COMPILER=${ROCM_HOME}/llvm/bin/clang++ \
+              onnxruntime_DEV_MODE=OFF \
```

Comment:
Yeah, will do it in latter PRs.

Currently, our ROCm build log is garbage filled with warnings which is quite annoying. This is enabled because hip clang is stricker at some point and cause build failure.